tools/flang1/flang1exe/lowersym.c

```diff
@@ -1221,13 +1221,96 @@ lower_pointer_init(void)
   }
 } /* lower_pointer_init */
 
+static void
```

Comment:
Can you put brief summary for the intent of this function ?

Reply:
A short description added. Please have a look.

Reply:
Could you elaborate? What is the function replacing the `FVAL` symbol and its descriptor symbol with? How are the `FVAL` symbol and the "dummy arguments" related to the `entry` parameter of function?